nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -399,11 +349,17 @@ contract StakingEscrow is Upgradeable, IERC900History {
     //------------------ ERC900 connectors ----------------------
 
     function totalStakedForAt(address _owner, uint256 _blockNumber) public view override returns (uint256) {
-        return stakerInfo[_owner].history.getValueAt(_blockNumber);
+        if (isUpgrade == UPGRADE_TRUE) {
+            return stakerInfo[_owner].history.getValueAt(_blockNumber);
+        }
+        return 0;
     }
 
-    function totalStakedAt(uint256 _blockNumber) public view virtual override returns (uint256) {
-        return balanceHistory.getValueAt(_blockNumber);
+    function totalStakedAt(uint256 _blockNumber) public view override returns (uint256) {
+        if (isUpgrade == UPGRADE_TRUE) {
+            return balanceHistory.getValueAt(_blockNumber);
+        }
+        return 0;
```

Comment:
For greater guarantees, what about continuing to return the `balanceHistory` here? That way voting power for individual addresses will be 0% for everyone. Otherwise, relative voting power (i.e., individual voting power / total voting power)will be 0/0, which I have no idea how Aragon would handle.
Alternatively, instead of 0, we could return here something like `type(uint256).max` or something like that.